spec/features/cancelling_a_per_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'feature_helper'
+
+RSpec.feature 'cancelling a PER', type: :feature do
+  scenario 'Cancelling a non issued PER' do
+    login
+
+    escort = create(:escort, :completed)
+
+    visit escort_path(escort)
+
+    escort_page.click_cancel
+
+    fill_in 'escort[cancelling_reason]', with: 'Started by mistake'
+    click_button 'Cancel this PER'
+
+    dashboard.assert_escorts_due(1)
```

Comment:
Should this also be asserted before the cancellation so that we know something actually changed?